dali/python/backend_impl.cc

```diff
@@ -1588,6 +1588,30 @@ void FeedPipeline(Pipeline *p, const string &name, py::list list, AccessOrder or
   p->SetExternalInput(name, tv, order, sync, use_copy_kernel);
 }
 
+struct PyPipeline: public Pipeline {
```

Comment:
PyBind doesn't accept it ;)

when we define init method for py:class_<A> it accepted std::unique_ptr<A> but didn't accpept unique_ptr with a deleter.

Reply:
I could define py::class_<Pipeline, std::unique_ptr<Pipeline, Deleter>>. It would accept such a ptr in init but,  I would have to change the way all the methods are exposed 
from 
```
.def("device_id", &Pipeline::device_id)
```
to
```
.def("device_id", [](Pipeline *p) { return p->device_id(); })
```
And that would be a lot of work